endpoints.json

```diff
@@ -39,6 +39,22 @@
     }
   },
 
+  "PATCH /api/articles/:article_id": {
+    "description": "increases the given article id's votecount as per the body provided in the request and returns an updated article",
+    "queries": "Article to be selected by article id within the URL",
+    "formatOfRequestBody": {"inc_votes": "NUMBER TO INCREASE BY"},
```

Comment:
For this you can simply give a number as an example instead just to show a real world use case for this endpoint.